types/src/v2/traits.rs

```diff
@@ -11,5 +11,5 @@ impl<R, T> RpcMethod<R> for T where T: Fn(RpcParams) -> Result<R, RpcError> + Se
 
 /// RPC Call Result.
 trait RpcResult {
-	fn to_json(self, id: Option<&RawValue>) -> anyhow::Result<String>;
+	fn into_json(self, id: Option<&RawValue>) -> anyhow::Result<String>;
```

Comment:
There are no impls of this trait anywhere, which is a bit odd. Do we need this trait @niklasad1 ?

Reply:
No, I removed it in another pending PR.

Just a nit when moving those types from `ws server`